Source/ACE/Entity/Player.cs

```diff
@@ -1371,14 +1376,17 @@ public bool GetVirtualOnlineStatus()
 
         private void SendSelf()
```

Comment:
just something I noticed in retail pcaps, but haven't investigated much... 

following the CreatePlayer and Player Description Event there is often a DeleteObject which deletes yourself followed by a CreateObject which creates yourself. The DeleteObject message isn't fully handled by aclogview so there could some something else in that packet which puts this in to better context

Not sure I get why, but worthing noting and investigating.